### ISSUE TITLE
[Accessibility]: Code intelligence policy radio buttons

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35184]
### Audit type

Screen reader navigation

### User journey audit issue

#33516

### Problem description

<img width="339" alt="radio buttons" src="https://user-images.githubusercontent.com/103087/167502993-9c092118-2c6d-4ec7-829e-87a963c4dd05.png">

The radio buttons on the code intelligence policy information is not intuitive with a screen reader. It has the same value for the button as the label, so it reads the label out redundantly.

### Expected behavior

Figure out what best practices are for radio buttons. See if @sourcegraph/frontend-platform has opinions or plans to do this site-wide.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by macraig** [comment]
Closing as identified issues have been completed
